staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -431,6 +431,10 @@ func (v *VirtualMachineInstance) IsBootloaderEFI() bool {
 		v.Spec.Domain.Firmware.Bootloader.EFI != nil
 }
 
+func (v *VirtualMachineInstance) IsMemoryHotplugEnabled() bool {
```

Comment:
You're right, that part is in virt-controller though, this function is only used in virt-launcher.

I'm actually going to remove it since it's used only in 1 place and it does not need to be public.